Address some feedback comments: notifications, test. And rename section

clemente opened the pull request:
This includes some comments from OC-3352.

# Testing
1. check the diff. HTML shouldn't be broken
2. Create .tar.gz and import it into Studio, into a course with the following data: course:„onboarding“, org:„OpenCraft“, run:„course“
3. follow the course. Check that there's a new section about notifications, and that the section „configuring your opencraft e-mail“ was renamed to a longer name
4. check that the internal link added in „first technical task“ to „code quality“ works
5. check the content, it must address:
  - types of tests, how to run each, some are slow
  - handle notification overflow; synchronous/asynchronous and times
  - over-commiting; how to take tasks progressively during the week
  - communication about non-task related work
  - and in addition, the issues requested as part of the review in github

pomegranited commented on html/configuration_of_the_opencraft_e-mail_mailing_lists_1.html:
```diff
@@ -1,7 +1,7 @@
+  <a href="https://github.com/open-craft/documentation/blob/master/mailing_lists.md" target="_blank">
    mailing-lists
   </a>
   and subscribe to the relevant mailing-lists with your @opencraft.com e-mail.
```
I think it's worth pulling this note from [mailing_lists.md](https://github.com/open-craft/documentation/blob/master/mailing_lists.md#opencraft-mailing-lists) of the /documentation and into this section:

> Note that if you are a newcomer you only need to subscribe to help@opencraft.com and urgent@opencraft.com if you need to access client communication in a task.

and noting the [Gmail_filters.md](https://github.com/open-craft/documentation/blob/master/Gmail_filters.md) reference too.

pomegranited commented on html/configuration_of_the_opencraft_e-mail_notifications_1.xml:
```diff
@@ -0,0 +1 @@
+<html filename="configuration_of_the_opencraft_e-mail_notifications_1"/>
```
Not sure where to put this request, so just adding it here..

I think the subsection name needs changing.  Could you change "Configuration of the OpenCraft e-mail"  to be "OpenCraft email, accounts, and mailing lists" ?

UmanShahzad commented on html/configuration_of_the_opencraft_e-mail_notifications_1.html:
```diff
@@ -0,0 +1,7 @@
+<ul>
+<li>Use Gmail to filter out emails, put them into folders, etc.</li>
+<li>Unsubscribe from JIRA tickets and Github notifications that do not relate to what you're working on.</li>
+<li>Disable notifications (such as your phone) while you are working.</li>
```
Can we indent all? 2 spaces preferred because I think that's what the HTML editor in Studio does by default.

UmanShahzad commented on html/first_technical_task_first_task_1.html:
```diff
@@ -49,6 +49,11 @@
+	See the unit about
+	<a href="/jump_to_id/opencraft_coding_standards" target="[object Object]">
+	  Coding standards
+	</a>
```
Hop into the HTML editor and try to fix this up manually, i.e. put it on the same line. Also, note the `target=[object Object]`.

clemente replied:
:-( I was doing the opposite transformation (*manually*), from a single line to the one-tag-per-line format, because the later is what I found in the git repository, and I wanted to preserve it. Ok, I'll try to reverse it, maybe the tinymce editor helps here.

`target="[object Object]"` is, ironically, the output that my Open edX produced when I created a new link using the default WYSIWYG editor. I suppose it makes no sense, so I'll change it to `"_blank"`.

UmanShahzad replied:
Yeah, when we had this style earlier, I was hoping that we could change it, but we were already 50h into the task so it didn't make sense to continue delaying delivery for those nits. We should aim to remove these spaces as we touch their lines for the task at hand, otherwise we'll continue to have awkward spaces before punctuation, which looks bad.

clemente replied:
No problem, I reindented all files to 2 spaces, and corrected the HTML to avoid multi-line <a>, I like this type of text changes. With GNU tools (including Emacs macros) it was easy, the hard part was to know that it was ok to do it, not the process itself.
I also fixed the target, &nbsp;, bogus <span>text</span>, …

UmanShahzad commented on html/whats_expected_opencraft_team_member_3.html:
```diff
@@ -6,88 +6,72 @@ <h4>
-    If I can't answer immediately, I will at least send a quick reply
-    to let them know we're working on a response.
+   <li>I will CC or forward to <a href=
+   "mailto:help@opencraft.com">help@opencraft.com</a> all client
```
I believe this linebreak will break the link itself.

UmanShahzad commented on vertical/opencraft_coding_standards.xml:
```diff
@@ -1,3 +1,4 @@
 <vertical display_name="Coding standards">
   <html url_name="opencraft_coding_standards_1"/>
+  <html url_name="opencraft_coding_standards_2"/>
```
Thanks for keeping the names rolling like this :)

UmanShahzad commented on vertical/configuration_of_the_opencraft_e-mail_notifications.xml:
```diff
@@ -0,0 +1,3 @@
+<vertical display_name="Notifications">
+  <html url_name="configuration_of_the_opencraft_e-mail_notifications_1"/>
```
Thanks for 2 spaces here :)

UmanShahzad commented on html/opencraft_coding_standards_2.html:
```diff
@@ -0,0 +1,13 @@
+<h4>Tests</h4>
+<p>Test coverage is our insurance policy against upstream changes, and so we are careful to ensure that the features we write are tested well. You can find the commands to run the different types of tests in our documentation:&nbsp;&nbsp;<a href="https://github.com/open-craft/documentation/blob/master/Code_quality_and_tests.md" target="[object Object]">Code quality and tests</a>.</p>
```
Nit: fix `target=[object Object]`.

UmanShahzad replied:
It would be useful to grep for this in case we miss any, before merging.

clemente commented on html/whats_expected_opencraft_team_member_3.html:
```diff
@@ -6,88 +6,72 @@ <h4>
    If any client email is addressed to me, I will respond to it within 24 hours
    (excluding weekend days), or ensure that someone else on the team does.
- </strong>
+  </strong>
```
The change was in wording, to mention that JIRA can take 1 day:


Old:
   <li>
    Generally, discussions should happen first asynchronously on the JIRA tickets;
    if there is really something that can't be efficiently sorted out asynchronously,
    have a chat or schedule a meeting. This will sometimes be necessary
    when long response cycles on JIRA mean that asynchronous messages
    wouldn't be able to unblock someone in time for them to finish their sprint commitments.
   </li>


New:
	  <li>Generally, discussions should happen first asynchronously
		on the JIRA tickets; if there is really something that can't be
		efficiently sorted out asynchronously, have a chat or schedule
		a meeting. JIRA might have long response cycles (around 1 day
		turnaround). If this time isn't enough to unblock someone and
		finish their sprint commitments, use Mattermost, even though
		it's more disruptive to people's workflow.</li>

clemente commented:
@pomegranited  @UmanShahzad I applied all your changes. Sorry that the diff is so verbose because of all HTML format fixes. You can check that the content is still the same, or import the course and check the content there.

clemente commented:
I noticed that are still some files that use TABs for indentation, and others spaces. I could transform each TAB to spaces if you want… or maybe it doesn't matter because this HTML will be overwritten if we ever export from Studio.

UmanShahzad commented:
@clemente Thanks for these changes, but that diff is huge and out of scope for review? I only mentioned to change the stuff we touch as part of the ticket, not everything in the repo.

To make @pomegranited's life easier, do you think you can move the commit that introduced the huge changeset to a new PR? (I'd review it and merge out of any ticket).

clemente commented:
@UmanShahzad or @pomegranited, I redid all my changes on top of the `onboarding` branch which now contains the HTML fixes. I rebased and fixed dozens of conflicts. You may check this final version; the changes are the same you saw earlier. You can merge if you want (or I'll do it), and someone should import this course to https://courses.opencraft.com/ or make me admin there (I'm registered with my e-mail).

pomegranited commented on html/open_edx_documentation_official_edx_documentation_1.html:
```diff
@@ -1,6 +1,6 @@
 <p>
   Please review the various parts of the
-  <a href="https://github.com/open-craft/documentation/blob/onboarding-feedback-10-17/official_documentation.md" target="_blank">Official edX documentation</a>
+  <a href="https://github.com/open-craft/documentation/blob/master/official_documentation.md" target="_blank">Official edX documentation</a>
```
👍  Thank you for fixing these!

pomegranited commented on html/time_management_and_invoices_tips_from_the_team_members_3.html:
```diff
@@ -25,6 +23,6 @@ <h3>
   </p>
   <p>
-    <a href="https://github.com/open-craft/documentation/blob/onboarding-feedback-10-17/images/braden-time-tracking.png" target="_blank"><img src="https://github.com/open-craft/documentation/raw/onboarding-feedback-10-17/images/braden-time-tracking.png" alt="" /></a>
+    <a href="https://github.com/open-craft/documentation/blob/master/images/braden-time-tracking.png" target="_blank"><img src="https://github.com/open-craft/documentation/raw/master/images/braden-time-tracking.png" alt="" /></a>
```
a11y nit: this image needs a descriptive alt tag, e.g.
> Screenshot of Braden's gmail with automatically-added tags.
